cli/src/commit_templater.rs

```diff
@@ -1445,7 +1446,42 @@ fn builtin_commit_evolution_entry_methods<'repo>()
             Ok(out_property.into_dyn_wrapped())
         },
     );
-    // TODO: add predecessors() -> Vec<Commit>?
+    map.insert(
+        "predecessors",
+        |_language, _diagnostics, _build_ctx, self_property, function| {
+            function.expect_no_arguments()?;
+            let out_property = self_property.and_then(|entry| {
+                let commits: Vec<_> = entry.predecessors().try_collect()?;
+                Ok(commits)
+            });
+            Ok(out_property.into_dyn_wrapped())
+        },
+    );
+    map.insert(
+        "inter_diff",
```

Comment:
nit: this could be called a `evolution.diff()`, but I don't feel strongly. `evolution.inter_diff()` might be clearer since there's also `evolution.commit().diff()`.